src/Commands/ClearCacheCommand.php

```diff
@@ -17,7 +15,7 @@ public function handle(): int
     {
         Translator::clearCache();
 
-        info('Cache cleared');
```

Comment:
Why note prompt `info` function?

Reply:
**Before:**

![image](https://github.com/user-attachments/assets/c49fad90-4e28-45c3-b7f1-9ece7f0cea53)

**Now:**

![image](https://github.com/user-attachments/assets/f7120242-8c52-4318-bf7f-2d9f96e9bd62)


For more highlighting the message.